src/NetworkPort_NetworkPort.php

```diff
@@ -309,7 +309,9 @@ public function storeConnectionLog($action)
             $netports_id = $this->fields['networkports_id_1'];
         } else {
             $netport->getFromDB($this->fields['networkports_id_2']);
-            if ($netport->fields['itemtype'] === NetworkEquipment::class) {
+            // Note: phpstan doens't understand that `$netport->getFromDB` will
+            // modify the content of `$netport->fields`
+            if ($netport->fields['itemtype'] === NetworkEquipment::class) { // @phpstan-ignore identical.alwaysFalse
```

Comment:
In fact, if the item cannot be loaded, `fields` will not be updated. The safer fix for this is to add add the `getFromDB()` result in the if conditions.

Reply:
Indeed. You can add a `$netport = new NetworkPort();` between `else {` and `$netport->getFromDB($this->fields['networkports_id_2']);` to make sure to reset the fields.